test/fiat-connect-client.test.ts

```diff
@@ -255,59 +309,76 @@ describe('FiatConnectClientImpl', () => {
         jest.advanceTimersByTime(2000) // timeout is 1000
         return JSON.stringify(mockQuoteInResponse)
       })
-      const response = await client.createQuoteIn(mockQuoteRequestQuery)
+      const response = await client._createQuote(
+        mockQuoteRequestQuery,
+        'in',
+        quoteResponseSchema,
+      )
 
       expect(response.isOk).toBeFalsy()
       expect(response.unwrap.bind(response)).toThrow(
         new ResponseError('AbortError'),
       )
     })
   })
-  describe('getQuoteOut', () => {
-    it('calls /quote/out and returns QuoteResponse', async () => {
-      fetchMock.mockResponseOnce(JSON.stringify(mockQuoteOutResponse))
-      const response = await client.createQuoteOut(mockQuoteRequestQuery)
-      expect(fetchMock).toHaveBeenCalledWith(
-        'https://fiat-connect-api.com/quote/out', // ?fiatType=USD&cryptoType=cUSD&country=DE
-        expect.objectContaining({
-          method: 'POST',
-          headers: { 'Content-Type': 'application/json' },
-          body: JSON.stringify(mockQuoteRequestQuery),
-        }),
-      )
-      expect(response.isOk).toBeTruthy()
-      expect(response.unwrap()).toMatchObject(mockQuoteOutResponse)
+  describe('createQuoteIn', () => {
+    it('calls _createQuote with the expected parameters', async () => {
+      client._createQuote = jest.fn().mockResolvedValue('quote')
```

Comment:
optional nit: you can do `jest.spyOn(client, '_createQuote').mockResolvedValueOnce('quote')` and avoid creating the client every time in beforeEach